library/alloc/src/fmt.rs

```diff
@@ -604,9 +604,14 @@ use crate::string;
 #[cfg(not(no_global_oom_handling))]
 #[must_use]
 #[stable(feature = "rust1", since = "1.0.0")]
+#[inline]
```

Comment:
Why this inline?

Reply:
Needs to be otherwise it won't optimise to a single to_owned